ml-agents/mlagents/trainers/trainer/trainer.py

```diff
@@ -42,9 +42,8 @@ def __init__(
         self.brain_name = brain_name
         self.run_id = run_id
         self.trainer_parameters = trainer_parameters
-        self.summary_path = trainer_parameters["summary_path"]
         self._threaded = trainer_parameters.get("threaded", True)
-        self._stats_reporter = StatsReporter(self.summary_path)
+        self._stats_reporter = StatsReporter(brain_name)
```

Comment:
Does this / how does this change the behavior of the StatsWriters?

Reply:
Previously the StatsReporter saved everything to the directory specified here, and `summary_path = {run_id}_{brain_name}` (no actual path, `summaries/` was assigned on construction of the StatsWriter in `learn.py`.). So changing to just `brain_name` puts it in `results/{brain_name}`.